src/librbd/api/Trash.cc

```diff
@@ -34,7 +34,7 @@ namespace librbd {
 namespace api {
 
 template <typename I>
-const typename Trash<I>::TrashImageSources Trash<I>::RESTORE_SOURCE_WHITELIST {
+const typename Trash<I>::TrashImageSources Trash<I>::RESTORE_SOURCE_ALLOWLIST {
```

Comment:
Nit: similar comment here:
```suggestion
const typename Trash<I>::TrashImageSources Trash<I>::ALLOWED_RESTORE_SOURCES {
```